### PR TITLE
Validation: description-missing fixes.

### PR DESCRIPTION
**Pull request by brittmsantos**
Added missing descriptions to fix validation errors.

### CODE REVIEW

**Comment by brittmsantos** [comment]
Added missing descriptions to fix validation errors.

**Comment by brittmsantos** [comment]
@stevemunk @rbrundritt

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

File: BingMaps/v8-web-control/modules/directions-module/distanceunit-enumeration.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "DistanceUnit Enumeration | Microsoft Docs"
+description: Describes DistanceUnit enumeration, which is related to unit distance, and provides a list of relevant options.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/directions-module/distanceunit-enumeration.md:3]:
```suggestion
description: Describes the DistanceUnit enumeration, which can be a distance defined in miles or kilometers.
```

File: BingMaps/v8-web-control/modules/directions-module/index.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Directions Module | Microsoft Docs"
+description: Describes the Directions module, which calculates directions, and provides API reference documentation and examples.
```
Comment (stevemunk) [inline BingMaps/v8-web-control/modules/directions-module/index.md:3]:
```suggestion
description: The overview page for the Directions Module section contains a description of the Microsoft.Maps.Directions API, which contains types that allow you to calculate route directions and display them on your Bing Maps V8 Control, and provides links to API reference articles and code examples.
```

**Review by stevemunk — CHANGES_REQUESTED** [review]
Please also make the following changes to the applicable descriptions:

Current: "and provides a list of properties"  or  "and provides a list of the object's properties" or "and provides a list of related properties"
Suggested: and provides descriptions for each of its properties

Current: and provides its constructor, methods, and events.
Suggested: and provides a description for its constructor, methods, and events.

**Comment by brittmsantos** [comment]
@stevemunk 

I've implemented the requested changes. Please let me know if there are any other issues preventing approval and merge. Thank you and have a great rest of your day!

**Pull request merged**